### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by kvraamkey**
new simple tab component added.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -35,23 +35,26 @@
   background: rgba(214, 214, 214, 0.5);
   border: none;
   letter-spacing: 1px;
-  padding: 14px;
-  margin-bottom: 5px;
-  height: 40px;
+  padding: 16px;
+  /* margin-bottom: 5px; */
```
Comment (madhuni) [inline src/App.css:39]: Can we please remove the comments?

File: src/App.css [hunk 2]
```diff
@@ -60,3 +63,31 @@
...
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```
Comment (madhuni) [inline src/App.css:93]: Please add the new line at the end of the file.

File: src/components/Tabs/Tabs.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    )
+}
+
+function TabItem(props) {
+    return <div {...props} />;
+}
+
+Tabs.Tab = TabItem;
+
+export default Tabs;
```
Comment (madhuni) [inline src/components/Tabs/Tabs.jsx:40]: Please add the new line.

**Comment by kvraamkey** [comment]
Can you merge?

**Comment by madhuni** [comment]
@kvraamkey, I don't have the permission to merge.

@NehemiahK can you please help with merging this PR?

**Pull request merged**